torchtext/experimental/datasets/text_classification.py

```diff
@@ -37,7 +36,7 @@ def _generate_data_iterators(dataset_name, root, ngrams, tokenizer, data_select)
     if not tokenizer:
         tokenizer = get_tokenizer("basic_english")
 
-    if not set(data_select).issubset(set(('train', 'test'))):
+    if not set(data_select).issubset({'train', 'test'}):
```

Comment:
Is this relevant to the drop of python2 support?

Reply:
Yes, it's using Python 3 syntax that was not possible when also supporting Python 2.